my-app/src/App.js

```diff
@@ -0,0 +1,67 @@
+import './App.css';
```

Comment:
1. React should be in scope
2. Use components to avoid duplication
3. Use "react" way of working with styles
Check out https://github.com/meltaway/dsc-studygroup-react/pull/1#discussion_r606843506 for example

Reply:
Also install some formatter to fix indentation and empty space